frontend/plugins/frontline_ui/src/modules/pipelines/components/PipelineDetail.tsx

```
@@ -0,0 +1,61 @@
+  const { updatePipeline } = useUpdatePipeline();
+
+  const form = useForm<IPipeline>({
+    resolver: zodResolver(PIPELINE_FORM_SCHEMA),
+  });
+  useEffect(() => {
+    form.reset({
+      name: pipeline?.name || '',
+      description: pipeline?.description || '',
+      _id: pipelineId || '',
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)